.circleci/config.yml

```diff
@@ -53,13 +53,30 @@ save-gems-cache: &save-gems-cache
   paths:
     - vendor/bundle
 
-update-fastlane: &update-fastlane
+update-fastlane-ios: &update-fastlane-ios
   name: Update Fastlane
   command: |
     echo "ruby-2.6.4" > ~/.ruby-version
-    bundle install
+    bundle install && bundle update
   working_directory: ios
 
+update-fastlane-android: &update-fastlane-android
+  name: Update Fastlane
+  command: |
+    echo "ruby-2.6.4" > ~/.ruby-version
+    bundle install && bundle update
+  working_directory: android
```

Comment:
Shouldn't this be a param?

Reply:
@diegolmello needs some more light on this. I have gone through their [official documentation](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration) but not able to get how to use it with anchors

Reply:
@diegolmello I have followed the above-suggested way. Looks like I am missing something.

Reply:
@GOVINDDIXIT It's ok. Rollback to the point it was working and move on.

Reply:
Sure 👍